src/app/modules/core/manager/reservation/components/details/manager-reservation-details.component.html

```diff
@@ -0,0 +1 @@
+<p>details works!</p>
```

Comment:
Please, rename the file, the component class, all the related files and the selector in `reservation-details` instead of `manager-reservation-details`. Most probably this component will be shared with the landlord module. Thanks.